### ISSUE TITLE
Training MS-Celeb-1M

### ISSUE THREAD

**Issue by xhzcyc** [wuqianliang/deepid2_caffe#4]
Hi, are you training ms-celeb-1m dataset now?
How long did you extract image from ms-celeb-1m.tsv file?

**Comment by wuqianliang** [comment]
I use the SSD disk; Extracting image from ms-celeb-1m.tsv may take a long while , about several hours. I only extract 20000 persons' images   for training. @xhzcyc

**Comment by xhzcyc** [comment]
How about your training result? @wuqianliang

**Comment by xhzcyc** [comment]
What's the size of your lmdb file that generated from 20000 person's image?

**Comment by wuqianliang** [comment]
deepid1  , test softmax accuracy  70%
deepid2 , test softmax accuracy  37%

DeepID1_train_lmdb  19G
DeepID1_test_lmdb   4.5G

This week , I will try multipatch preprocess on train dataset.  @xhzcyc

**Comment by xhzcyc** [comment]
Could you share your train_lmdb & test_lmdb ?

**Comment by xhzcyc** [comment]
Why are your test accuracy so low?
I have trained 10000 people subset and get a good result!

**Comment by wuqianliang** [comment]
Could you share the sovler txt for deepid1 and deepid2 ? @xhzcyc 
my emailbox :    wuqianliang@ismartearth.com

**Comment by wuqianliang** [comment]
Do you have a wechat ?  @xhzcyc

My wechat is 15715164075

**Comment by Yodamt** [comment]
用的deepid2跑了四万次 准确率63是不是有点低了
I0305 19:09:06.048393 22020 solver.cpp:337] Iteration 40000, Testing net (#0)
I0305 19:09:47.081173 22020 solver.cpp:404]     Test net output #0: accuracy1 = 0.634328
I0305 19:09:47.081173 22020 solver.cpp:404]     Test net output #1: sim_loss = 0.0132883 (* 0.3 = 0.00398649 loss)
I0305 19:09:47.081173 22020 solver.cpp:404]     Test net output #2: softmax_loss = 2.3179 (* 0.7 = 1.62253 loss)

**Comment by Yodamt** [comment]
> 用的deepid2跑了四万次 准确率63是不是有点低了
> I0305 19:09:06.048393 22020 solver.cpp:337] Iteration 40000, Testing net (#0)
> I0305 19:09:47.081173 22020 solver.cpp:404] Test net output #0: accuracy1 = 0.634328
> I0305 19:09:47.081173 22020 solver.cpp:404] Test net output #1: sim_loss = 0.0132883 (* 0.3 = 0.00398649 loss)
> I0305 19:09:47.081173 22020 solver.cpp:404] Test net output #2: softmax_loss = 2.3179 (* 0.7 = 1.62253 loss)

用的是lfw-aligned的train.txt 一共三千左右样本

**Comment by wuqianliang** [comment]
> > 用的deepid2跑了四万次 准确率63是不是有点低了
> > I0305 19:09:06.048393 22020 solver.cpp:337] Iteration 40000, Testing net (#0)
> > I0305 19:09:47.081173 22020 solver.cpp:404] Test net output #0: accuracy1 = 0.634328
> > I0305 19:09:47.081173 22020 solver.cpp:404] Test net output #1: sim_loss = 0.0132883 (* 0.3 = 0.00398649 loss)
> > I0305 19:09:47.081173 22020 solver.cpp:404] Test net output #2: softmax_loss = 2.3179 (* 0.7 = 1.62253 loss)
> 
> 用的是lfw-aligned的train.txt 一共三千左右样本

建议你用sphereface,效果比较好。
LFW数据集太小，效果相对会比较差一些。
